wagtailio/packages/models.py

```diff
@@ -0,0 +1,154 @@
+from django.contrib.postgres.fields import ArrayField
+from django.db import models
+from django.db.models import F
+
+from wagtail.admin.edit_handlers import (
+    FieldPanel,
+    MultiFieldPanel,
+)
+from wagtail.core.models import Page
+
+from wagtailio.utils.models import CrossPageMixin, SocialMediaMixin
+
+
+class PackagesPage(Page, SocialMediaMixin, CrossPageMixin):
+    def get_context(self, request, *args, **kwargs):
+        context = super().get_context(request, *args, **kwargs)
+        context.update(
+            {
+                "grids": Grid.objects.filter(publish=True)
+                .order_by(F("order").desc(nulls_last=True), "title")
+                .prefetch_related("packages")
+            }
+        )
+        return context
+
+
+class Package(models.Model):
+    """
+      {
```

Comment:
The first line of a multi-line of the docstring should be a summary (see pep 257).

Might also be worth mentioning that the JSON doc is an example response from djangopackages API?